Off-Top-App/off-top-flutter
# OT-MigrateFlutterEndpoints | Migrate flutter to be hitting our newly created microservices

**WHAT I DID:**
1. Migrated flutter to be hitting our newly created microservices. 
2. Also updated the User object so that it passes the properly named attribute of `profession` instead of `professional`
3.  No longer passing `password` to the backend anymore as it is not required in the User microservice and no longer exists as a column in the user MySQL table

HOW TO TEST:
1. Run Kafka/Zookeeper, [UserService](https://github.com/Off-Top-App/UserService) on master branch, & [SessionHandler](https://github.com/Off-Top-App/SessionHandlerService/tree/OT-SessionHandler). For SessionHandler make sure you're on branch OT-SessionHandler. For this branch, make sure you're on branch OT-MigrateFlutterEndpoints
2. Make sure that you can create an account and can login to the application. Because you are now hitting endpoints from a different microservice when dealing with account creation and login.
3. Make sure that you can send audio data to the WebSocket which now lives in SessionHandlerService microservice
4. Make sure you can Producer.py in [off-top-python](https://github.com/Off-Top-App/off-top-python)/backend-services/Services/Kafka/Producer.py
    - You will need to change the user_id on line 23 to user_id value associated within your MySQL DB of offTopUser.
